recipes/libgsf/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -euo pipefail
+
+# fix perl shebang
+sed -i.bak '1 s|^.*$|#!/usr/bin/env perl|g' $SRC_DIR/tests/t*.pl
+
+./configure --prefix=${PREFIX}
+make
```

Comment:
You might compile it in parallel
```suggestion
make -j${CPU_COUNT}
```

Reply:
Not the part that is most time consuming, but will add it. Thanks!